spec/lib/triannon-client/triannon-client_spec.rb

```diff
@@ -23,24 +24,105 @@ def create_annotation
     }
   end
 
+  def delete_annotation(id)
+    tc = TriannonClient::TriannonClient.new
+    tc.delete_annotation(id)
+  end
+
   def check_graph_has_statements(graph)
     expect(graph).to be_instance_of RDF::Graph
-    expect(graph.empty?).to be_falsy
-    expect(graph.size > 2).to be_truthy
+    expect(graph).not_to be_empty
+    expect(graph.size).to be > 2
   end
 
   before :all do
     Dotenv.load
-    @tc = TriannonClient::TriannonClient.new
     @oa_jsonld = '{"@context":"http://iiif.io/api/presentation/2/context.json","@graph":[{"@id":"_:g70349699654640","@type":["dctypes:Text","cnt:ContentAsText"],"chars":"I love this!","format":"text/plain","language":"en"},{"@type":"oa:Annotation","motivation":"oa:commenting","on":"http://purl.stanford.edu/kq131cs7229","resource":"_:g70349699654640"}]}'
   end
 
-  before :each do
-    @anno = create_annotation
+  # before :each do
+  #   @anno = create_annotation
+  # end
+
+  # after :each do
+  #   tc = TriannonClient::TriannonClient.new
+  #   tc.delete_annotation(@anno[:id])
+  # end
+
+  describe 'has constant' do
+    it 'CONTENT_TYPES' do
+      const = TriannonClient::TriannonClient::CONTENT_TYPES
+      expect(const).to be_instance_of Array
+      expect(const).to include('application/ld+json')
+    end
+    it 'CONTENT_TYPE_IIIF' do
+      const = TriannonClient::TriannonClient::CONTENT_TYPE_IIIF
+      expect(const).to be_instance_of String
+      expect(const).to include('application/ld+json')
+      expect(const).to include('profile="http://iiif.io/api/presentation/2/context.json"')
```

Comment:
I wouldn't spend a lot of time on it, but if your constants are Strings, checking that they are eq "asdfasdfasd" makes the first of these tests redundant.    I'm not sure I would even bother with these tests, but since you've got 'em you might as well leave 'em in.